CHANGELOG.md

```diff
@@ -199,6 +199,13 @@
 - Nightly test failure: spgemm unit tests failing on White \(Power8\) [\#780](https://github.com/kokkos/kokkos-kernels/issues/780)
 - supernodal does not build with UVM enabled [\#633](https://github.com/kokkos/kokkos-kernels/issues/633)
 
+## [3.1.01](https://github.com/kokkos/kokkos-kernels/tree/3.1.01) (2020-05-04)
```

Comment:
We change the change logs with an automated process before release.

Reply:
Sorry I shouldn't have committed either of these (master_history or CHANGELOG). It was an artifact of merging/reflogging my slightly outdated branch into the current develop. I'll revert these to the current 3.5 ones.

Reply:
Should be reverted now.